@google-cloud/storage/bucket/index.d.ts

```diff
@@ -0,0 +1,102 @@
+/// <reference types="node" />
+
+import { ReadStream } from "fs";
+import { Channel, ChannelConfig } from "../channel";
+import { File } from "../file";
+import {
+  Acl,
+  ApiResponse,
+  UploadOptions
+} from "../storage";
+
+declare module "@google-cloud/storage/bucket" {
```

Comment:
@blove Question: why you declare ambient module here instead of just declare "export class Bucket ..."etc at the top level? Is it that the library is meant to use with "require" as well